tests/bench_thread_yield_pingpong/main.c

```diff
@@ -22,7 +22,7 @@
 
 #include "macros/units.h"
```

Comment:
This test is still missing the includes for `CLOCK_CORECLOCK`

```suggestion
#include "periph_conf.h"
#include "timex.h"
#include "macros/units.h"
```